<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Massimiliano Torromeo</author>
 <class>SQLAntaresiaWindow</class>
 <widget class="QMainWindow" name="SQLAntaresiaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SQL Antaresia</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/32/kexi.png</normaloff>:/32/kexi.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cmbConnection">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabsWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Server">
    <property name="title">
     <string>Se&amp;rver</string>
    </property>
    <addaction name="actionReconnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_SQLAntaresia"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionShow_Databases"/>
    <addaction name="actionShow_Statusbar"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure_Connections"/>
    <addaction name="actionConfigure_SQLAntaresia"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Server"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBarConnection">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Databases</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTreeView" name="treeView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionRefresh_Table_List">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/view-refresh.png</normaloff>:/16/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Table List</string>
   </property>
  </action>
  <action name="actionRefresh_Databases">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/view-refresh.png</normaloff>:/16/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Databases</string>
   </property>
  </action>
  <action name="actionDrop_Table">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/edit-delete.png</normaloff>:/16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Drop Table</string>
   </property>
  </action>
  <action name="actionClose_Tab">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/tab-close.png</normaloff>:/16/tab-close.png</iconset>
   </property>
   <property name="text">
    <string>Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShow_Statusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show St&amp;atusbar</string>
   </property>
  </action>
  <action name="actionConfigure_SQLAntaresia">
   <property name="text">
    <string>Configure &amp;SQL Antaresia</string>
   </property>
  </action>
  <action name="actionConfigure_Connections">
   <property name="text">
    <string>Configure &amp;Connections</string>
   </property>
  </action>
  <action name="actionAbout_SQLAntaresia">
   <property name="text">
    <string>About SQL Antaresia</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/application-exit.png</normaloff>:/16/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionShow_Databases">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Databases</string>
   </property>
  </action>
  <action name="actionDrop_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/edit-delete.png</normaloff>:/16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Drop Database</string>
   </property>
   <property name="toolTip">
    <string>Drop Database</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="text">
    <string>&amp;Reconnect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/network-disconnect.png</normaloff>:/16/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionTruncate_Table">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/edit-delete.png</normaloff>:/16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Truncate Table</string>
   </property>
  </action>
  <action name="actionNew_Query_Tab">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16/tab-new.png</normaloff>:/16/tab-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Query Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>tabsWidget</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SQLAntaresiaWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Statusbar</sender>
   <signal>triggered(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_Databases</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShow_Databases</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
